distributed/worker.py

```diff
@@ -2254,12 +2261,12 @@ def transition_cancelled_released(
 
         for resource, quantity in ts.resource_restrictions.items():
             self.available_resources[resource] += quantity
-        recs, instructions = self.transition_generic_released(
-            ts, stimulus_id=stimulus_id
+
+        return merge_recs_instructions(
+            self.transition_generic_released(ts, stimulus_id=stimulus_id),
+            ({ts: next_state} if next_state != "released" else {}, []),
+            self._ensure_computing(),
```

Comment:
If we call `_ensure_computing` as part of `transition_generic_released` we no longer need this here, do we?

Reply:
I assume calling it twice is harmless. I'm OK with keeping it. I actually don't like it being part of `transition_generic_released` but there is probably more work hidden so I'm good

Reply:
removed